testresults/src/org/labkey/testresults/SendTestResultsEmail.java

```diff
@@ -180,18 +205,18 @@ private String getBackgroundStyle(BackgroundColor color) {
                         if (isGoodRun)
                             goodRuns++;
 
-                        message.append("<tr style='border-bottom:1px solid grey;'>");
-                        message.append("\n<td style=\"" + style + " padding: 6px;\"><a href=\"" +
-                                "http://skyline.ms" + new ActionURL(TestResultsController.ShowRunAction.class, container) + "runId=" + run.getId()
+                        message.append("<tr style='border-bottom:1px solid grey;'>")
+                            .append("\n<td style=\"" + style + " padding: 6px;\"><a href=\"" +
+                                "https://skyline.ms" + new ActionURL(TestResultsController.ShowRunAction.class, container) + "runId=" + run.getId()
```

Comment:
If you need an absolute URL, you can simply call `getURIString()` on `ActionURL`. Also best to use `addParameter()` instead of adding parameters via String concatenation. i.e., `new ActionURL(TestResultsController.ShowRunAction.class, container).addParameter("runId", Integer.valueOf(run.getId())).getURIString()`. (On the develop branch, the parameter could be added with `addParameter("runId", run.getId())`.)